vortex-array/src/arrays/list/compute/is_constant.rs

```diff
@@ -0,0 +1,134 @@
+use vortex_error::VortexResult;
+
+use crate::arrays::{ListArray, ListVTable};
+use crate::compute::{IsConstantKernel, IsConstantKernelAdapter, IsConstantOpts};
+use crate::register_kernel;
+
+impl IsConstantKernel for ListVTable {
+    fn is_constant(&self, array: &ListArray, opts: &IsConstantOpts) -> VortexResult<Option<bool>> {
+        // At this point, we're guaranteed:
+        // - Array has at least 2 elements
+        // - All elements are valid (no nulls)
+
+        if opts.is_negligible_cost() {
+            return Ok(None);
+        }
+
+        let first_list_len = array.offset_at(1) - array.offset_at(0);
```

Comment:
Can you do something like `array.offsets().is_constant()`?

Reply:
we need `offsets.diff().is_constant()` here right? We need some sort of `scan` reduction compute function to do this I think